discord/command.go

```diff
@@ -385,12 +385,13 @@ type StringChoice struct {
 
 // IntegerOption is a subcommand option that fits into a CommandOptionValue.
 type IntegerOption struct {
-	OptionName  string          `json:"name"`
-	Description string          `json:"description"`
-	Required    bool            `json:"required"`
-	Min         option.Int      `json:"min_value,omitempty"`
-	Max         option.Int      `json:"max_value,omitempty"`
-	Choices     []IntegerChoice `json:"choices,omitempty"`
+	OptionName   string          `json:"name"`
+	Description  string          `json:"description"`
+	Required     bool            `json:"required"`
+	Min          option.Int      `json:"min_value,omitempty"`
+	Max          option.Int      `json:"max_value,omitempty"`
+	Choices      []IntegerChoice `json:"choices,omitempty"`
+	Autocomplete bool            `json:"autocomplete"`
```

Comment:
I think we should document that `Autocomplete must not be true if Choices are present.`